README.md

```diff
@@ -56,7 +62,7 @@ jobs:
           # RSS feed URL
           RSS_URL: "https://hnrss.org/newest"
           TWEET_TEMPLATE: 'New Post: "%title%" %url%'
-          UPDATE_WITHIN_MINUTES: 15 # for workflow_dispatch
+          UPDATE_WITHIN_MINUTES: ${{ inputs.updateMinutes }} # for workflow_dispatch
```

Comment:
This value seems to be empty when executed in `on.schedule`.
Therefore, I think that 60 minutes of hand-held clay will be used.
https://github.com/azu/rss-to-twitter/blob/aff320f38e4916c3c9c8f303410b5a6721be97ba/src/process.ts#L62